libs/base/shims.d.ts

```diff
@@ -2,8 +2,20 @@
 
 
 
-//% indexerGet=BufferMethods::getByte indexerSet=BufferMethods::setByte
+    //% indexerGet=BufferMethods::getByte indexerSet=BufferMethods::setByte
 declare interface Buffer {
+    /**
+     * Reads an unsigned byte at a particular location
+     */
+    //% shim=BufferMethods::getUint8
+    getUint8(off: int32): int32;
+
+    /**
+     * Writes an unsigned byte at a particular location
+     */
+    //% shim=BufferMethods::setUint8
+    setUint8(off: int32, v: int32): void;
```

Comment:
Why do we need these? Indexers should just work and otherwise getNumber

Reply:
The indexers don't work for jacdac-ts when we cross-compile with regular typescript.